langchain/llms/__init__.py

```diff
@@ -36,6 +37,7 @@
     "DeepInfra",
     "ForefrontAI",
     "GooseAI",
+    "GPT4All",
```

Comment:
What do you think about testing __all__ for changes? (unrelated to this PR obviously)

Reply:
what do you mean?